llpc/util/llpcShaderModuleHelper.cpp

```diff
@@ -96,6 +96,9 @@ ShaderModuleUsage ShaderModuleHelper::getShaderModuleUsageInfo(const BinaryData
       if (decoration == DecorationInvariant) {
         shaderModuleUsage.useInvariant = true;
       }
+      if (decoration == DecorationBuiltIn && static_cast<BuiltIn>(codePos[4]) == BuiltInPointSize) {
```

Comment:
Shouldn't the index `4` here depend on whether the opcode is OpDecorate or OpMemberDecorate? If this builtin can only appear in a struct, please at least add an assertion to that effect.

Reply:
Yes, it should depend on OpMemberDecorte. I missed the condition. Thanks!